openmc/source.py

```diff
@@ -131,6 +143,8 @@ def to_xml_element(self):
             element.set("particle", self.particle)
         if self.file is not None:
             element.set("file", self.file)
+        if self.library is not None:
```

Comment:
It looks like right now a source can have a full "native" definition as well as a custom library attribute. Which of those will take preference when exporting to XML or, more importantly, when initializing a simulation? Might be something for a follow-up PR.